altsrc/toml_file_loader.go

```diff
@@ -0,0 +1,113 @@
+// Disabling building of toml support in cases where golang is 1.0 or 1.1
+// as the encoding library is not implemented or supported.
+
+// +build go1.2
+
+package altsrc
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/BurntSushi/toml"
+	"github.com/urfave/cli"
+)
+
+type TomlMap struct {
```

Comment:
Is it required to have this type be public? It seems that its just used internally and its not needed to be public, but not sure if the underlying deserializer requires a public container type